test/test_models.py

```diff
@@ -66,16 +70,18 @@ def _test_classification_model(self, name, input_shape):
         num_classes = 50
 
         # Pretrained parameters
-        self.assertRaises(ValueError, models.__dict__[name], pretrained=True, imagenet_pretrained=True)
-
-        # Default case
-        model = models.__dict__[name](num_classes=num_classes)
-        model.eval()
-        x = torch.rand(input_shape)
-        with torch.no_grad():
-            out = model(x)
-        # self.assertExpected(out, rtol=1e-2, atol=0.)
-        self.assertEqual(out.shape[-1], 50)
+        args = inspect.getfullargspec(models.__dict__[name])
+        if 'imagenet_pretrained' in args.args:
```

Comment:
This could be kept if passing imagenet_pretrained actually passes pretrained=True to holocron function